andela/ah-django-unchained-frontend
# #162948794 set an empty react app

**What does this PR do?**
- It creates a new react application

**Description of Task to be completed?**
- Creating a new application using `npm create-react-app .`

**How should this be manually tested?**
- clone this repo `https://github.com/andela/ah-django-unchained-frontend`
- `cd ah-django-unchained-frontend`
- Install the required dependencies `npm install`
- `npm start to run the local server`
- The following message show that your server has been started successfully
 ``` 
You can now view ah-django-unchained-frontend in the browser.
  Local:            http://localhost:3004/
  On Your Network:  http://10.6.54.151:3004/

Note that the development build is not optimized.
To create a production build, use npm run build. 
```
**Pivotal Tracker story**

[#162948794](https://www.pivotaltracker.com/story/show/162948794)
